test/parser/au_splitter_test.exs

```diff
@@ -3,9 +3,11 @@ defmodule AUSplitterTest do
 
   use ExUnit.Case
 
-  alias Membrane.H264.FFmpeg.Parser.Native, as: NativeParser
-
   @test_files_names ["10-720a", "10-720p"]
+  @au_lengths_ffmpeg %{
```

Comment:
A short comment explaining that these are fixtures obtained from the old parser or sth like this would be appreciated

Reply:
done